### DISCUSSION TITLE
NexoPOS 4.7.1

### DISCUSSION THREAD

**Discussion by Blair2004** [Blair2004/NexoPOS#663] [Releases]
This update provide few bug fixes over the core and ensures Gastro works well with NexoPOS 4.7.x. We're keep updating all module to make sure they are all compatible with NexoPOS 4.7.x. Until that moment, if you're using a module that remain on version v4.6.x, do not upgrade to NexoPOS 4.7.x now.

Changelog:
========
- Fixed: product missing when category is missing
- Updating: Crud instances
- Added: new test for paying with customer credit

<hr /><em>This discussion was created from the release <a href='https://github.com/Blair2004/NexoPOS-4x/releases/tag/v4.7.1'>NexoPOS 4.7.1</a>.</em>

**Comment by osvaldoclasico** [comment]
Good evening, when creating a new client it no longer allows me to add the phone number. Do you have any idea why it doesn't work?
Link Visual
https://ibb.co/cQQmy9x

**Reply by Blair2004** [reply]
Thank you for pointing that out. We've spotted a wrong validation rule.

**Reply by osvaldoclasico** [reply]
Thanks friend, I will be aware of the other updates, thanks for taking into account my past recommendations, the new dark theme and the other modifications are very good
